src/lamp_py/performance_manager/alerts.py

```diff
@@ -489,22 +485,35 @@ def process_alerts(md_db_manager: DatabaseManager) -> None:
     """
     process_logger = ProcessLogger("process_alerts")
     process_logger.log_start()
-    unprocessed_records = get_unprocessed_alert_files(md_db_manager)
+
+    version_match = version_check(
+        obj=AlertsS3Info.s3_path, version=AlertsS3Info.file_version
+    )
```

Comment:
check for a version match. if regenerating, we'll pull _all_ files from springboard bucket and we won't pull historical record from s3.